src/types/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface ExampleProps {
```

Comment:
We usually define component props in the same file where. component is defined. For easier tracking and having all in one place related to component